src/util/tryOr.ts

```diff
@@ -1,7 +1,7 @@
-export default function tryOr<T>(fn: () => T): T | undefined {
+export default function tryOr<T, E = undefined>(fn: () => T, elseReturn?: E): T | E | undefined {
```

Comment:
I don't think this change is used anymore.

Reply:
Reverted